docs/RELEASE-NOTES.rst

```diff
@@ -1,38 +1,55 @@
-    - Enable this support using deployment parameter `hubmode:true`
-* Helm Chart Enhancements:
-   - Added ingressClass resource installation with charts
-* Added support for AS3 3.28
+* CIS now compatible with:
+    - Kubernetes 1.21
+    - OpenShift 4.7.13 with OpenShift SDN
+    - AS3 3.28
 
+* Added support for:
+   - :issues: `1824` Ingress and IngressClass objects in networking.k8s.io/v1. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/ingress/networkingV1>`_.
+   - Multiport Service and Health Monitor for Service Type LB in CRD mode. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/crd/serviceTypeLB>`_.
 
-Bug Fixes
-`````````
-* :issues: `1824` Support for ingresses.networking.k8s.io/v1.
-* Fixed rewrite-url annotation doesnt support to rewrite all child paths to specific target domain
+* New CIS deployment configuration options:
+  - --periodic-sync-interval - Configure the periodic sync of resources.
+  - --hubmode:true - Enable Support for ConfigMaps to monitor services in same and different namespaces.
+  - --disable-teems=true|false - Configure to send anonymous analytics data to F5 TEEM infrastructure.
 
-* Fixed http redirect not working when used virtual-server.f5.com/rewrite-target-url annotation with routes
-* Fixed inconsistent behaviour with CIS (Transport Server) while adding/deleting a node to cluster.
+* CIS now monitors changes to Kubernetes Secret resource.
+* Improved performance while processing Ingress resources.
+* CIS in AS3 agent mode now adds default cipher groups to SSL profiles for TLS v1.3.
+* CIS in AS3 agent mode now supports OpenShift Routes with annotations `rewrite-app-root` and `rewrite-target-url`. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/docs/config_examples/ingress/networkingV1/sample-url-rewrite-annotation.yaml>`_.
+  The user need to create ingressClass resource for CIS before upgrading to CIS v2.5.0 while using ingress.
+* CIS now supports FIC with InfoBlox (Preview).
+* :issues: `1570` `1571` Added support for iRule reference in TransportServer CRD.  Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/crd/TransportServer>`_.
 
-Limitations
-```````````
-* Due to networking.k8s.io/v1 api support in ingress customer has to use "virtual-server.f5.com/clientssl" annotation in ingress if they are using bigip profiles in tls spec of ingress resource.
+
+Helm Chart Enhancements
+````````````````````````
+* Added support to install Ingress and IngressClass objects in networking.k8s.io/v1
+
+
+Bugs Fixes
+``````````
+* CIS now properly adds nodes as pool members (in NodePort mode).
+
+
+Known Issues
+````````````
+* Recommendation to configure CIS deployment with --periodic-sync-interval more than 300 seconds for improved performance. OpenShift Routes with termination Passthrough get processed post this interval.
```

Comment:
For improved performance, it is recommended to configure CIS deployment with --periodic-sync-interval​more than 300 seconds. OpenShift Routes with termination Passthrough are processed post this interval.